integration/common/openlineage/common/provider/dbt.py

```diff
@@ -633,9 +633,21 @@ def extract_namespace(self, profile: Dict) -> str:
             return "bigquery"
         elif profile['type'] == 'redshift':
             return f"redshift://{profile['host']}:{profile['port']}"
+        elif profile['type'] == 'spark':
+            if profile['method'] in ['thrift', 'odbc', 'http']:
+                port = ""
```

Comment:
_Minor_: Should we provide a default port if not specified by the user?

Reply:
Done. Those are different depending on method: `thrift` is `10001`, `odbc` and `http` are `443`.